app/(frontend)/auth/login/page.tsx

```diff
@@ -29,9 +50,10 @@ export default function LoginPage() {
             />
 
             <div className="flex flex-col justify-center items-center gap-4 lg:col-span-2 lg:row-span-2">
+
                 <h1 className="text-4xl font-bold">Login</h1>
 
-                {/*<AuthForm handleSubmit={handleSubmit}/>*/}
+                {typeof state === "string" && <p className="text-red-500">{state}</p>}
```

Comment:
In the JSX code, other than the JSX, only function calls should be there. If you would need to modify this, it would take much of your time to find this line in the code. Refactor it into its own function inside this component. This issue will be referred as **Refactor into function**